### DISCUSSION TITLE
Can this handle GPGSA and GPRMC Data?

### DISCUSSION THREAD

**Discussion by Riprock** [semuconsulting/pynmeagps#63] [Q&A] [answered]
Can this handle GPGSA and GPRMC Data? I want to be able to take the GPGSA and GPRMC and route it elsewhere. I already have the logic in place for the 3 different data types. I just need to know if this can currently parse this data

**Comment by semuadmin** [comment, marked as answer]
Yes. From the README…

The library implements a comprehensive set of outbound (GET) and inbound (SET/POLL) GNSS NMEA messages relating to GNSS/GPS devices, but is readily [extensible](https://github.com/semuconsulting/pynmeagps/discussions/63#extensibility). Refer to [NMEA_MSGIDS](https://github.com/semuconsulting/pynmeagps/blob/master/src/pynmeagps/nmeatypes_core.py#L172) and [NMEA_MSGIDS_PROP](https://github.com/semuconsulting/pynmeagps/blob/master/src/pynmeagps/nmeatypes_core.py#L224) for the complete dictionary of standard and proprietary messages currently supported.

**Comment by Riprock** [comment]
Thanks Sorry
